sirbot-pyslackers/endpoints/apscheduler.py

```diff
@@ -80,18 +80,37 @@ def create_jobs(scheduler, bot):
 async def etc_finance_bell(bot, state):
     LOG.info("Posting opening bell to #etc_finance")
```

Comment:
Very small nitpick that I missed on your previous PR. Since we now pass the state the log should reflect it and not always `opening`.